src/Select.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import sys from 'system-components'
 import { themeGet } from 'styled-system'
 
-const arrow = encodeURI(`data:image/svg+xml,<svg xmlns='http://www.w3c.org/2000/svg' width='32' height='32' viewbox='0 0 32 32' fill='tomato'> <path d='M0 6 L32 6 L16 28 z' /> </svg>`)
+const arrow = encodeURIComponent(`<svg xmlns='http://www.w3.org/2000/svg' width='100' height='100'><polygon points='0,0 100,0 50,50'/></svg>`)
```

Comment:
I'd rather keep the original SVG here, since that doesn't seem to be the issue this PR is intended to solve

Reply:
done, feel free to edit the PR 🙂